packages/@uppy/tus/src/index.js

```diff
@@ -55,7 +64,6 @@ module.exports = class Tus extends Plugin {
 
     // set default options
     const defaultOptions = {
-      resume: true,
```

Comment:
In tus-js-client v2 the `resume` option has been removed. When previously enabled, the uploader would look into localStorage to see if a file with the same fingerprint has been uploaded before and then resume from that URL. Regardless of whether `resume` is enabled, the retries of errored requests is controlled by the `retryDelays` option.

In v2, the app can now query the previous uploads on its own and ask the user (or determine on its own) which upload should be resumed. The API looks like this:

```js
upload = new tus.Upload(file, options);
  upload.findPreviousUploads().then((previousUploads) => {
    // Prompt user which upload should be resumed
    const index = askToResumeUpload(previousUploads);
    if (index != null) {
       upload.resumeFromPreviousUpload(previousUploads[index]);
    }

    upload.start();
  });
```

I didn't add the call to findPreviousUploads yet because I don't really know where to fit it. Maybe you can help?

Reply:
BTW, if you want the same behavior as the `resume: true` option, you can do following:

```
  upload = new tus.Upload(file, options);
  upload.findPreviousUploads().then((previousUploads) => {
    if (previousUploads.length > 0) {
       upload.resumeFromPreviousUpload(previousUploads[0]);
    }

    upload.start();
  });
```

Reply:
There isn't really a great place to put it until we actually support ghost files 😇 
I think the only thing we _can_ do is have a `resume: true` option in the Tus plugin that implements the old behaviour.

Reply:
Ok, thanks for the information. Would you be able to do that on your own since I am not really comfortable with developing Uppy? :sweat_smile: